auditai/cmh.py

```diff
@@ -0,0 +1,125 @@
+from scipy import stats
+from functools import partial
+import numpy as np
+import pandas as pd
+
+def extract_data(df):
+    # utility function for cmh test and odds ratio
+    # df needs to be a 2x2 data frame with columns 'pass' and 'fail'
+    a = float(df.iloc[0]['pass'])
+    d = float(df.iloc[1]['fail'])
```

Comment:
some slightly more illustrative variable names would be preferable throughout

Reply:
fixed!